modules/videopress/videopress.php

```diff
@@ -731,6 +759,33 @@ function videopress_shortcode_options( $options ) {
 		return $options;
 	}
 
+	/**
+	 * Get the upload api path.
+	 *
+	 * @param int $blog_id The id of the blog we're uploading to.
+	 * @return string
+	 */
+	protected function make_media_upload_path( $blog_id ) {
+		return sprintf(
+			'%s://%s/rest/v%s/videos/%s/new',
+			'https',
```

Comment:
Mostly consistency with how this is done in other parts of Jetpack. In this case, we are making requests from the browser, so we need to require that the calls are made over https.